test/on_replica_by_default_test.rb

```diff
@@ -55,29 +55,40 @@
   end
 
   it "executes `find` on the replica" do
-    account = Account.find(1000)
-    assert_equal "Replica account", account.name
+    with_all_primaries_unavailable do
```

Comment:
would you prefer to nest all tests within a `with_all_primaries_unavailable` block, or specify this block in each test? happy either way.

Reply:
It's nice to have access to the primary databases during test setup/teardown. And I think it's more explicit this way.